DANDI.md

```diff
@@ -0,0 +1,61 @@
+# DANDI: Distributed Archives for Neurophysiology Data Integration
```

Comment:
@tgbugs provide list of data modalities to @satra

Reply:
@tgbugs - let me know if there is a specific way to encode this.